cli/parser/parser.go

```diff
@@ -1069,6 +1069,23 @@ func GetBazelCommandAndIndex(args []string) (string, int) {
 	return "", -1
 }
 
+// GetBazelPatterns makes a best-attempt effort to return the pattern in a bazel command.
```

Comment:
`GetFirstTargetPattern` maybe? (`Patterns` sort of indicates that it would return a `[]string` with each pattern - there can be multiple patterns in a command)